### ISSUE TITLE
Milestone extension workflow and messages

### ISSUE THREAD

**Issue by gondzo** [appirio-tech/connect-app#2231]
Current milestone extension workflow is the same for all milestone types (Request extension/approve extension) and messages always refer to "low number of submissions". We should have custom messages for extending submission phases vs review extension. Alternative is to disable Request Extension button when we're waiting on client review

![image](https://user-images.githubusercontent.com/3051646/43138293-a455a606-8f4e-11e8-83c6-3c9daa9720eb.png)

@RishiRajSahu what do you think?

**Comment by rmjmccormick** [comment]
@vikasrohit Agree. Rather than a new button, couldn't we consider a toggle button instead for `Mark for Review`? It would be less invasive than a new `Recall Review` button

@gondzo In your milestone extension requested language above, please amend (third line) "...extend the milestone **with** 24h" to "...extend the milestone **by** 24h"

**Comment by vikasrohit** [comment]
Toggle button makes an impression that it is very common to make it toggle i.e. manager might take it very casual which ideally should not be because it changes the customer's view every time a manger toggles it. So, IMO, we should have new button in place of `Mark for Review` button as soon as milestone is marked for review. Essentially, customer would see it as just change of the text of the button from `Mark for Review` to `Recall Review`.

**Comment by rmjmccormick** [comment]
@vikasrohit As you describe it makes sense. We'll need to think it through a bit more though. Let's set up another issue for this as you originally suggested.

**Comment by gondzo** [comment]
OK, closing this one
